sdk/cosmos/azure-cosmos/azure/cosmos/_change_feed/feed_range_internal.py

```diff
@@ -33,6 +33,11 @@
 
 class FeedRangeInternal(ABC):
 
+    def __init__(self, container_link: str) -> None:
+        if container_link is None:
+            raise ValueError("container_link cannot be None")
+        self._container_link = container_link
```

Comment:
why we will need container_link in the feedRange?

Reply:
removed container_link